test/wasi/test-wasi.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+require('../common');
+
+const cp = require('child_process');
+const fs = require('fs');
+const path = require('path');
+const assert = require('assert');
+
+const SKIP = [
+  'poll',
+];
+
+const C_DIR = path.resolve(__dirname, 'c');
+const WASM_DIR = path.resolve(__dirname, 'wasm');
+
+// TODO(devsnek): remove this when V8 supports
+// bigint on non-x64.
```

Comment:
I might be wrong, but doesn’t the presence of e.g. `parallel/test-process-hrtime-bigint` imply that this is already the case?

Reply:
v8 only supports converting wasm's `i64` with a BigInt on x64. BigInt itself is supported everywhere v8 is.

Reply:
Might be worth updating the comment with that information then?